pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -618,6 +618,7 @@ func defaultOpenAPIConfig(config configapi.MasterConfig) *openapicommon.Config {
 			},
 		}
 	}
+	// TODO: handle WebhookTokenAuthenticators somehow ?
```

Comment:
probably just as an additional condition in `if len(config.ServiceAccountConfig.PublicKeyFiles) > 0 || ... token webhooks ... {`

Reply:
yeah I wasn't sure about that I'll do as you suggest